tests/board_test.py

```diff
@@ -0,0 +1,36 @@
+from ..lib.board import *
+
+
+class TestBoard:
+    def test_move(self):
+        #     move method changes the value of the given key( number from the user) from empty to the given mark
```

Comment:
Instead of having a comment here, we should define the method name to something more descriptive. Perhaps something like `test_move_marks_the_board_with_marker`, for example.